kcalign/__init__.py

```diff
@@ -575,23 +580,35 @@ def genome_mode(reference, reads, start, end, compress, para, tab):
         try:
             start = int(start)-1
             end = int(end)
-        except Exception:
-            print('User Error: invalid start/end coordinate(s)')
+        except ValueError:
```

Comment:
Also note that the try/except won't catch negative values, if that's the intention.